packages/go/ecosystem/ecosystem.go

```diff
@@ -0,0 +1,29 @@
+package ecosystem
+
+import (
+	"strings"
+
+	"github.com/ossf/osv-schema/constants"
+)
+
+type Parsed struct {
+	Ecosystem constants.Ecosystem
+	Suffix    string
+}
+
+func (p *Parsed) String() string {
```

Comment:
ah good question - I will make sure we've got tests in place that ensure ultimately this gets JSON'd as a string rather than an object, and that it gets parsed back like that too